src/Dev/TestMailer.php

```diff
@@ -41,7 +41,13 @@ public function send(RawMessage $message, Envelope $envelope = null): void
         /** @var Email $email */
         $email = $message;
         $this->dispatchEvent($email, $envelope);
-        $this->emailsSent[] = [
+        $data = $this->createData($email);
+        $this->emailsSent[] = $data;
```

Comment:
Since the `$data` variable is not used but once, how about...
```suggestion
        $this->emailsSent[] = $this->createData($email);
```

Or is it preparation for some further work?

Reply:
Updated